internal/trivy/scan_job.go

```diff
@@ -73,11 +74,22 @@ func (f *filesystemScanJobBuilder) ForCIS(cis *stasv1alpha1.ContainerImageScan)
 		stasv1alpha1.LabelK8SAppManagedBy:             stasv1alpha1.AppNameImageScanner,
 		stasv1alpha1.LabelStatnettControllerNamespace: cis.Namespace,
 		stasv1alpha1.LabelStatnettControllerUID:       string(cis.UID),
+		stasv1alpha1.LabelStatnettWorkloadKind:        cis.Spec.Workload.Kind,
+		stasv1alpha1.LabelStatnettWorkloadName:        workloadLabelName(cis),
+		stasv1alpha1.LabelStatnettWorkloadNamespace:   cis.Namespace,
 	}
 
 	return job, nil
 }
 
+func workloadLabelName(cis *stasv1alpha1.ContainerImageScan) string {
```

Comment:
I think we should make a simple utility function to truncate strings. I DON'T understand why Go does not have this fundamental thing as part of the language???